app-sync.js

```diff
@@ -139,9 +139,16 @@ app.post('/sync', async (req, res) => {
   for (let i = 0; i < newMessages.length; i++) {
     let msg = newMessages[i];
     let envelopePb = new SyncPb.MessageEnvelope();
+    let messagePb = new SyncPb.Message();
+
+    messagePb.setDataset(msg.dataset);
+    messagePb.setRow(msg.row);
+    messagePb.setColumn(msg.column);
+    messagePb.setValue(msg.value);
+
     envelopePb.setTimestamp(msg.timestamp);
-    envelopePb.setIsencrypted(msg.is_encrypted === 1);
-    envelopePb.setContent(msg.content);
+    envelopePb.setContent(messagePb.serializeBinary());
```

Comment:
To support the `sync-simple.js` use case, where that flow that stores the binary-encoded messages directly, let's move encoding the messages into `sync-full.js`. `newMessages` should be an array of message envelopes, so the loop just becomes:

```
  for (let msg of newMessages) {
    responsePb.addMessages(msg);
  }
```

We'll also need to update the `sync-simple.js` flow to encode the messages, but I can do that